pkg/ako/values.go

```diff
@@ -407,14 +407,17 @@ func DefaultControllerSettings() *ControllerSettings {
 
 // NewControllerSettings returns a ControllerSettings from default,
 // allow setting CloudName, ControllerIP, ControllerVersion and ServiceEngineGroupName
-func NewControllerSettings(cloudName, controllerIP, controllerVersion, serviceEngineGroup string) (setting *ControllerSettings) {
+func NewControllerSettings(cloudName, controllerIP, controllerVersion, serviceEngineGroup, tenantName string) (setting *ControllerSettings) {
 	setting = DefaultControllerSettings()
 	setting.CloudName = cloudName
 	setting.ControllerIP = controllerIP
 	setting.ServiceEngineGroupName = serviceEngineGroup
 	if controllerVersion != "" {
 		setting.ControllerVersion = controllerVersion
 	}
+	if tenantName != "" {
```

Comment:
we don't need this if statement for now since `tenantName` is default to be empty